docs/community-ee/community-ee-announcement.md

```diff
@@ -1,17 +1,7 @@
-# Announcing the community-ee-* execution environments
-
-We announce both community-ee-base and community-ee-minimal together.
-
-## Forum Announcement
+# Forum Announcement
```

Comment:
For the sentence below this, you can add a direct link to the correct place in the forum so no one makes a mistake on where to post it.  The link is https://forum.ansible.com/c/news/releases/18

Reply:
Added this line. 
```
Follow [forum link](https://forum.ansible.com/c/news/releases/18) to create the announcement post.
```